doc/AUTHORS.adoc

```diff
@@ -85,16 +90,15 @@ ccache is a collective work with contributions from many people, including:
 * Per Nordlöw
 * Peter Budai
 * Philippe Proulx
+* Philipp Storz
 * Rafael Kitover
 * Ramiro Polla
 * Robert Yang
 * Robin H. Johnson
 * Rolf Bjarne Kvinge
-* Russell King
 * RW
 * Ryan Brown
 * Sam Gross
-* Steffen Dettmer
```

Comment:
This means that the script doesn't pick up Steffen Dettmer from f2481b1d5988cb4ccc3010c4ad6933e9c93e6d02. I'll have a look.

Reply:
Fixed in b20f862db0ffcd16991663d56d69c22db5abfd5a.